### PR TITLE
chore(pet-119): archive spec artifacts to DONE/

### PR DESCRIPTION
**Pull request by ziomancer**
Docs-only follow-up to land the `/spec-close` archival for **PET-119** (the fix itself merged in #97).

Moves the PET-119 spec lifecycle artifacts from `docs/specs/TODO/` to `docs/specs/DONE/PET-119/`:

- `brief.md`, `spec.md`, `reconciliation.md`
- `reviews/round-1/{correctness,edge-cases,conventions}.md`
- `test-output.txt` (renamed into the DONE tree)

No code change. Matches the archival pattern from #47 ("retire shipped specs to DONE/"). This commit existed locally on `master` (created by `/spec-close`) but had not yet landed on origin; the `branch-protect` ruleset requires it to go through a PR.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Documentation**
  * Added comprehensive PET-119 specification materials, including a spec brief, reconciliation report, and round-1 review notes (conventions, correctness, and edge-case analysis).
  * Documented the new scan-time behavior for profile-driven entity selection via `profile.pii_entities_extra`, including additive opt-in semantics, effective entity contract (with order-preserving dedup for built-in profiles), and guarantees for default/no-profile behavior.
  * Included acceptance criteria, regression test requirements, and CI guidance to validate the change end-to-end.

### CODE REVIEW

File: docs/specs/DONE/PET-119/brief.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+### Effective entity contract (the semantics to implement)
+
+For a given scan under `active_profile`:
+
+```
+effective_entities = resolve_presidio_entities(
+    config.presidio_entities, config.presidio_entities_extra
+) ∪ active_profile.pii_entities_extra        # order-preserving dedup
+```
```
[the remark this answers is not in the document]
Comment (ziomancer) [reply docs/specs/DONE/PET-119/brief.md]: Resolved in efbe935

File: docs/specs/DONE/PET-119/reconciliation.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+| Test | Exists? | Location |
+|---|---|---|
+| no-extra uses base (negative pin) | Yes | `tests/test_presidio_entity_scoping.py::test_scan_no_extra_uses_base` |
+| extra_entities additive | Yes | `…::test_scan_extra_entities_additive` |
+| extra_entities dedup | Yes | `…::test_scan_extra_entities_dedup` |
+| no-mutation (Decision 6) | Yes | `…::test_scan_does_not_mutate_self_entities` |
+| unknown extra threads through (Presidio-is-the-filter) | Yes | `…::test_scan_unknown_extra_threads_through` |
+| pipeline passes profile extras to Presidio | Yes | `…::test_pipeline_passes_profile_extras_to_presidio` |
+| pipeline no-profile / general-empty use base | Yes | `…::test_pipeline_no_profile_uses_base`, `…::test_pipeline_general_profile_empty_extras_uses_base` |
```
[the remark this answers is not in the document]
Comment (ziomancer) [reply docs/specs/DONE/PET-119/reconciliation.md:63]: Skipped — `edge-cases.md` is a frozen round-1 review snapshot whose purpose is to record what that review found at that point in time. The resolution (the test now exists at `tests/test_presidio_entity_scoping.py:364`) is already documented in `reconciliation.md` (which this finding agrees is correct and needs no change). Backfilling resolution status into the historical review artifact would falsify the point-in-time record; reconciliation.md is the correct home for it.
